docs/ota-client-guide/modules/ROOT/pages/setup-boot-image-for-ostree.adoc

```diff
@@ -0,0 +1,16 @@
+= Set up boot image layout for OSTree compatibility

```

Comment:
Let's use the NXP board as a specific example here. You can re-write the page with the actual example already; we'll worry about linking to the commits later.

You can link to the PDF of the manual online, but it would also be good to screenshot the table with the info, just so we have something to go on if NXP decides to remove or change the location of that file.